snippets/AutoLink.md

```diff
@@ -16,7 +16,7 @@ function AutoLink({ text }) {
         if (match) {
           let url = match[0];
           return (
-            <a href={url.startsWith("http") ? url : `http://${url}`}>{url}</a>
+            <a href={url.startsWith("http") || url.startsWith("https") ? url : `http://${url}`}>{url}</a>
```

Comment:
`http` is a subset of `https` and thus will match with `https` since `(http)s`, as such this is redundant

```js
'https'.startsWith('http') // true
```